### PR TITLE
Minor REPL enhancements

### PR DESCRIPTION
**Pull request by tranma**
* .icicle file!!!
* handle ctrl-c (Max would be proud)

### CODE REVIEW

**Comment by jystic** [comment]
:100:

wondering if it would make sense to load an `.icicle` file from the current directory as well?

**Comment by tranma** [comment]
oh yeah, also should make it optional so if the user doesn't have a .icicle file, it's still good

File: main/icicle-repl.hs [hunk 1]
```diff
@@ -61,18 +61,22 @@ main
...
-             , autoAddHistory = True}
+    settings home
+      = return $ HL.defaultSettings
+          { historyFile    = Just $ home <> "/.icicle-repl.history"
+          , autoAddHistory = True}
+    dotfile home
+      = lines <$> readFile (home <> "/.icicle")
     loop state
-      = do line <- HL.getInputLine "> "
+      = handleInterrupt (loop state)
```
Comment (jystic) [inline main/icicle-repl.hs]: already better than `sbt` :laughing:

Comment (tranma) [inline main/icicle-repl.hs]: it still doesn't handle ctrl-c??

**Comment by jystic** [comment]
:honey_pot:

**Pull request merged**